lib/sensors/include/UltraSonicSensor.h

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+#include "Pwmln.h"
+#include "mbed.h"
+
+// declare constants
+const int NUM_SAMPLES{1};
```

Comment:
rather than having this as a constant, you could have this be a parameter thats configured on instantiation of this class. this way, you can say I want this instance of ultrasonic class to always make readings that are averaged over x number of samples rather than applying that at a class-wide level